autopilot-dashboard/app/autopilot-dashboard/values.yaml

```diff
@@ -0,0 +1,126 @@
+# Default values for autopilot-dashboard.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  repository: quay.io/anish2sinha/autopilot-dashboard
```

Comment:
Can you push the image to quay.io/autopilot?

Reply:
It should be fixed now. The image is pushed with tag "dashboard". The yaml now points to quay.io/autopilot/autopilot with tag "dashboard".